website/source/docs/providers/chef/r/data_bag.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Creates and manages a data bag in Chef Server.
 ---
 
-# chef\_data\_bag
+# chef_data_bag
```

Comment:
Yea, it only happens when there are spaces, so _foo_ is italic, but foo_bar_baz is not. See, even github got it right!